### ISSUE TITLE
Broken support for external address to favicon

### ISSUE THREAD

**Issue by stgraber** [evildmp/vanilla-sphinx-test#25]
Starting with Sphinx 4.0, the favicon can be an external URL, however the theme hardcodes `{{ pathto('_static/' + favicon, 1) }}` causing `http://SERVER/_static/https://...` to be accessed instead.

**Comment by anthonydillon** [comment]
@ru-fu Is this really required for a POC?

**Comment by ru-fu** [comment]
@anthonydillon Yes, because for LXD, we cannot put the favicon into the repo that has the docs because of license reasons.

But this should be a simple fix (in  https://github.com/evildmp/vanilla-sphinx-test/blob/8f45b0d52f61f28257047e3b6add805d333268f9/vanilla/base.html#L33 ). Let me know if I should make a PR.

**Comment by anthonydillon** [comment]
Please do, PRs always welcome.

I was under the impression this was only for a private/internal POC. Is that right?

**Comment by ru-fu** [comment]
Fixed with https://github.com/evildmp/vanilla-sphinx-test/pull/38

**Comment by petesfrench** [comment]
@evildmp Can you close this issue please
